_posts/2022-12-08-developer-survey-2022.md

```diff
@@ -0,0 +1,123 @@
+
+Reflecting upon the previous survey, in 2020 Kotlin was the number 1 most wanted in this question, with a fraction of a percentage of use in customer projects. In this survey, while it is still on the wantlist, Kotlin has already made it to top 10 most used languages in projects, and it is regularly applied in backend, frontend and other (e.g. scripting) contexts. Likewise, it is likely that in the next survey, Go and Rust will have a noticeable presence in the project languages response.
+
+Individual votes were also cast for Elixir, Haskell, Smalltalk, Elm, Lisp in the wanted section.
+
+[![Top wanted](/img/developer-survey-2022/chalkboard.png)](/img/developer-survey-2022/chalkboard.png)
+
+In the development environments, Visual Studio Code has bumped the IntelliJ Idea from the top position, and the vim/emacs derby is still turning for vim. Note that the illustration shows the percentage of votes per editor in a multiple choice question. A lot of our developers use multiple IDEs in their work with different projects, or within one for different contexts.
+
+[![IDE](/img/developer-survey-2022/screen.png)](/img/developer-survey-2022/screen.png)
+
+For their work computer OS, 52% choose Windows, 40% go for MacOS and 8% prefer Linux. The most popular Linux distribution used in Solita linux workstations is Ubuntu. Arch Linux and Debian get a handful of votes. 
+
+## Stacks
+
+Let's have a closer look to what did we do in the past two years then. In the following illustration you can see what were the most used languages in our projects. 
+
+[![Used languages](/img/developer-survey-2022/pen.png)](/img/developer-survey-2022/pen.png)
+
+Despite the rise of nosql solutions, oldschool relational databases are still in the majority, hence SQL in the lead. Among the databases we use most are PostgreSQL, Microsoft SQL Server, Azure SQL, Oracle, SQLite and MySQL. 
+
+We do a lot of frontend development for our customers, so it is natural to see a lot of JavaScript used. Compared to 2020, TypeScript has jumped up a couple of positions, and is almost neck to neck with JavaScript now. In the backend section, C# and plain old Java are holding the fort, but Python and Kotlin are running up. Solita has always been known for Clojure knowhow, and it is expected to stay high on the list also in the future. Clojure is used in both backend and frontend work at Solita.
+
+In the frameworks compartment, for backend we are using .NET Core and ASP.NET with the .NET runtime, Spring Boot and Spring Framework with JVM, Express/node.js and Flask/Python.
+
+For frontend development, the framework ruling the set at Solita is react with almost half of the votes (45%). The runner-ups are vue.js (14%) and Angular (13%). From the clojure context, Reagent and re-frame are also getting around 10% of votes each.
```

Comment:
We could capitalize "clojure context" to "Clojure context".

Reply:
fixed